apps/util-func/src/domain/message-metadata.ts

```diff
@@ -0,0 +1,3 @@
+export interface MessageMetadataDeleter {
+  deleteMessageMetadata: (partitionKey: string, id: string) => Promise<void>;
+}
```

Comment:
I think that these deleter should not be placed here - the domain known only about messages and should not act on their part independently

Reply:
[70778d8](https://github.com/pagopa/io-messages/pull/181/commits/70778d8bfe656be318d01855b0a5f16bd5e0b189)